File: pset1/mario-more/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int a = 0;
    do
    {
        a = get_int("altura: ");
    }
    while (a <= 0 || a > 8);

    for(int i = 1; i <= a; i++)
    {
        for (int e = 0; e < a - i; e++)
        {
            printf(" ");
        }
        for (int c = 0; c < i; c++)
        {
            printf("#");
        }
        printf("  ");
        for (int b = 0; b < i; b++)
        {
            printf("#");
        }
        printf("\n");
    }
    return 0;
}

File: pset6/sentimental-mario-less/mario.py
from cs50 import get_int

while  True:
    a = get_int("height: ")
    if a >= 1 and a <= 8:
        break

for e in range(a):
    print((a - 1 - e) * " ", end= "")
    print((e + 1) * "#")